Example/Tests/XCTestCase+XCTestCase_JSZVCRAdditions.m

```diff
@@ -56,7 +56,25 @@ - (NSURLRequest *)performUniqueVerifiedNetworkCall:(void (^)(NSURLRequest *reque
     return request;
 }
 
+- (void)performLargeChunked:(int) numBytes verifiedNetworkCall:(void (^)(NSData *, NSURLResponse *, NSError *))extraVerifications {
+    NSString *url = [NSString stringWithFormat:@"https://httpbin.org/drip?numbytes=%i&duration=0&code=200", numBytes];
+    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:[NSURL URLWithString:url]];
+    [self performNetworkRequest:request withTimeout:50 verification:^(NSData *data, NSURLResponse *response, NSError *error) {
+        XCTAssertNil(error);
+        XCTAssertNotNil(response);
+        XCTAssertNotNil(data);
+        XCTAssertEqual(data.length, numBytes);
+        if (extraVerifications) {
+            extraVerifications(data, response, error);
+        }
+    }];
+}
+
 - (void)performNetworkRequest:(NSURLRequest *)request withVerification:(void (^)(NSData *, NSURLResponse *, NSError *))verifications {
+    [self performNetworkRequest:request withTimeout:5 verification:verifications];
+}
+
+- (void)performNetworkRequest:(NSURLRequest *)request withTimeout:(int)timeout verification:(void (^)(NSData *, NSURLResponse *, NSError *))verifications {
```

Comment:
The larger requests take longer, so I added a timeout parameter so we could slow down that one test case without affecting the others.